frontend/src/metabase/containers/AddToDashSelectDashModal.tsx

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable import/order */
+/* eslint-disable react/prop-types */
```

Comment:
Could we remove both of these "disable" comments? Prop types should be fine now and order should be easy to fix